static/katex.less

```diff
@@ -49,6 +49,18 @@
         display: inline-block;
     }
 
+    .mathrm {
+        font-style: normal;
+    }
+
+    .textit {
+        font-style: italic;
+    }
+
+    .textsc {
+        font-variant: small-caps;
+    }
```

Comment:
Since `small-caps` doesn't have super wide support yet we should probably leave this out or make it optional for now.  A more robust way of implementing this would be to use CSS transforms to scale normal caps.  Unfortunately, transforms don't affect the flow and so then have to adjust for the gaps left.  :(